api/tests/Feature/TrainingOpportunityTest.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Enums\CourseFormat;
+use App\Enums\CourseLanguage;
+use App\Models\TrainingOpportunity;
+use App\Models\User;
+use Database\Seeders\ClassificationSeeder;
+use Database\Seeders\RolePermissionSeeder;
+use Illuminate\Foundation\Testing\Concerns\InteractsWithExceptionHandling;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Nuwave\Lighthouse\Testing\RefreshesSchemaCache;
+use Tests\TestCase;
+use Tests\UsesProtectedGraphqlEndpoint;
+
+class TrainingOpportunityTest extends TestCase
+{
+    use InteractsWithExceptionHandling;
+    use MakesGraphQLRequests;
+    use RefreshDatabase;
+    use RefreshesSchemaCache;
+    use UsesProtectedGraphqlEndpoint;
+
+    protected $admin;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->bootRefreshesSchemaCache();
+        $this->withoutExceptionHandling();
+        $this->seed(ClassificationSeeder::class);
+        $this->seed(RolePermissionSeeder::class);
+
+        $this->admin = User::factory()
+            ->asApplicant()
+            ->asAdmin()
+            ->create([
+                'email' => 'admin@test.com',
+                'sub' => 'admin@test.com',
+            ]);
+    }
+
+    public function testQueryingPaginatedDeadlineScope(): void
+    {
+        TrainingOpportunity::truncate();
```

Comment:
Is this not handled by `RefreshDatabase`?

Reply:
Force of habit, but redundant here. 
Removed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12010/commits/9198ed5340e8c85ec706e09c6747650fc21bb27d